src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -23,6 +23,36 @@ def __init__(self, node: TraversalNode, endpoints: Dict[str, Endpoint]):
         super().__init__(node)
         self.endpoints = endpoints
 
+    @staticmethod
+    def _build_request_body(  # pylint: disable=R0913
+        custom_body: Optional[str],
+        path: str,
+        param_name: str,
+        default_value: str = None,
+        field_reference: str = None,
+        identity: str = None,
+    ) -> Optional[str]:
+        """Method to build request body based on config vals. Common to both read and update requests."""
+        if not custom_body:
+            logger.info(f"Missing custom body {path}")
+            return None
+        if default_value:
+            custom_body = custom_body.replace(f"<{param_name}>", f'"{default_value}"')
+        elif field_reference:
```

Comment:
If both `default_value` and `field_reference` are provided we'll ignore this field, we should mention this in the docstring

Reply:
We do have a nice validator `check_exactly_one_value_field` in `main` that will prevent both `default_value` and `field_reference` from being defined, so this scenario is unlikely. Should I still update the docstring?